.travis.yml

```diff
@@ -36,6 +39,10 @@ matrix:
     - rvm: jruby-9.1.9.0
     - rvm: ruby-head
     - env: "CHILDPROCESS_POSIX_SPAWN=true"
+  include:
+  - rvm: jruby-9.2.5.0
+    jdk: openjdk11
+    env: "JAVA_OPTS_FOR_SPECS='--add-opens java.base/java.io=org.jruby.dist --add-opens java.base/sun.nio.ch=org.jruby.dist'"
```

Comment:
I might be missing a nuance of how `.travis.yml` treats `env`, but could you remove the `before_script` and just have the following in your `include`:

```yaml
env:
  - "JAVA_OPTS=--add-opens java.base/java.io=org.jruby.dist --add-opens java.base/sun.nio.ch=org.jruby.dist"
```

...or does that not work for some reason? I ask because it would avoid setting `JAVA_OPTS=""` for all other builds.

Reply:
Several of the setup commands that run through jruby (and therefore create a JVM) do not need those options, so this was an attempt to only provide them to the invocation that runs rspec. I am open to other options.